context/ethereumContext.tsx

```diff
@@ -157,7 +159,10 @@ export const EthereumProvider: React.FC<{}> = ({ children }) => {
       chain: Chain.Mainnet,
       hardfork: fork || CURRENT_FORK,
     })
-    vm = new VM({ common })
+
+    vm = await VM.create({ common })
+
+    evm = await EVM.create({ common, eei: vm.eei })
```

Comment:
this is interesting that there is no a single evm instance and we have to use both. Is this something coming from ethereumjs?

Reply:
So that one is a bit annoying, but I couldn't find a better way around it.
We use `evm`'s `getActiveOpcodes` function in our component. The problem with it, is that since `@ethereumjs`'s latest version had separated the `evm` and `vm` classes, it is no longer possible to access this function using `vm`.
Even trying something like `vm.evm.getActiveOpcodes` isn't working, so I had to resort to instantiating an `evm` object and access the function this way.

Reply:
Can you eventually open a short issue on this? This is something we should definitely address.

Reply:
@holgerd77 sure thing. It's [here](https://github.com/ethereumjs/ethereumjs-monorepo/issues/2363).